README.md

```diff
@@ -32,11 +32,14 @@ Follow these steps to build an AMI for GraphDB using Packer:
    instance type, VPC ID, and subnet ID. To do so, create a variables file `variables.pkrvars.hcl`, example file: 
 
    ```bash
-   gdb_version  = "10.3.2"
+   gdb_version  = "10.3.3"
    build_aws_regions = ["eu-central-1"]
-   build_instance_type = "m5.large"
    build_vpc_id = "<your-vpc-id>"
    build_subnet_id = "<your-subnet-id>"
+   build_instance_type_x86-64    = "m5.large"
```

Comment:
Check if t4.small works for us, as discussed

Reply:
t3.small is working